src/context/TastStoreContextProvider.tsx

```diff
@@ -1,35 +1,64 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
 import { createContext, useState } from 'react';
-import { TASKS_STORAGE_KEY } from '../constants/storage';
+import { TASKS_LIST_STORAGE_KEY, TASK_GROUPS_STORAGE_KEY } from '../constants/storage';
 import { createLocalStorageStore } from '../tools/sotrage';
 import { TaskItemType } from '../types/TaskItemType';
+import { TaskItemGroupType } from '../types/TaskItemGroupType';
 
 export type TaskStoreContextType = {
+  taskGroupList: TaskItemGroupType[];
+  setTaskGroupList: React.Dispatch<React.SetStateAction<TaskItemGroupType[]>>;
+  localStorageTaskGroupStore: {
```

Comment:
let's keep one structure for taskStorage, do not create separate storage for task groups and taskList inside

Reply:
done with TaskStoreType, which contains groups and tasksList